README.md

```diff
@@ -14,16 +14,21 @@ A set of tools to analyze genomic data with a focus on Next Generation Sequencin
 Detailed user documentation is available on the [project website](http://fulcrumgenomics.github.io/fgbio/) including [tool usage](http://fulcrumgenomics.github.io/fgbio/tools/latest) and [documentation of metrics produced](http://fulcrumgenomics.github.io/fgbio/metrics/latest).  Detailed developer documentation can be found [here](http://javadoc.io/doc/com.fulcrumgenomics/fgbio_2.13).
 
 <!---toc start-->
-  * [Goals](#goals)
-  * [Overview](#overview)
-  * [List of tools](#list-of-tools)
-  * [Building](#building)
-  * [Command line](#command-line)
-  * [Include fgbio in your project](#include-fgbio-in-your-project)
-  * [Contributing](#contributing)
-  * [Authors](#authors)
-  * [License](#license)
-  * [Sponsorship](#sponsorship)
+- [fgbio](#fgbio)
+- [Goals](#goals)
+  - [Overview](#overview)
+  - [List of tools](#list-of-tools)
+  - [Building](#building)
+    - [Cloning the Repository](#cloning-the-repository)
+    - [Running the build](#running-the-build)
+  - [Command line](#command-line)
+  - [Include fgbio in your project](#include-fgbio-in-your-project)
+  - [Contributing](#contributing)
+  - [Authors](#authors)
+  - [License](#license)
+  - [Sponsorship](#sponsorship)
+    - [Become a sponsor](#become-a-sponsor)
+    - [Sponsors](#sponsors)
```

Comment:
GitHub has TOC support so we could consider removing this maintenance burden.

https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

Reply:
VS code also automatically updates the TOC. I don't care either way.